src/open_inwoner/conf/base.py

```diff
@@ -370,9 +383,22 @@
             "level": "INFO",
             "propagate": True,
         },
+        "requests": {
+            "handlers": ["log_outgoing_requests", "save_outgoing_requests"],
+            "level": "DEBUG",
+            "propagate": True,
+        },
     },
 }
 
+
+#
+# LOG OUTGOING REQUESTS
+#
+LOG_OUTGOING_REQUESTS_ENABLED = config("LOG_OUTGOING_REQUESTS_ENABLED", default=False)
```

Comment:
I think we might not need `LOG_OUTGOING_REQUESTS_ENABLED`? It now just works like `LOG_OUTGOING_REQUESTS_DB_SAVE` (and the stream logging is now controlled by LOGGING)